apps/files/src/services/DropService.ts

```diff
@@ -67,11 +70,18 @@ export const handleDrop = async (data: DataTransfer): Promise<Upload[]> => {
 const handleFileUpload = async (file: File, path: string = '') => {
 	const uploader = getUploader()
 
-	try {
-		return await uploader.upload(`${path}${file.name}`, file)
-	} catch (e) {
-		showError(t('files', 'Uploading "{filename}" failed', { filename: file.name }))
-		throw e
+	const forbidden = forbiddenCharacters.split('')
+	let forbiddenChar
+
+	if(forbiddenChar = forbidden.find(char => file.name.includes(char))){
+		showError(t('files', `"${forbiddenChar}" is not allowed inside a file name.`));
+	}else{
```

Comment:
same here about our code style
```suggestion
	} else {
```